src/topology/algebra/monoid.lean

```diff
@@ -85,6 +77,19 @@ lemma continuous_within_at.mul [topological_space β] {f : β → α} {g : β 
   continuous_within_at (λx, f x * g x) s x :=
 hf.mul hg
 
+@[to_additive]
+instance [topological_space β] [has_mul β] [has_continuous_mul β] : has_continuous_mul (α × β) :=
+⟨((continuous_fst.comp continuous_fst).mul (continuous_fst.comp continuous_snd)).prod_mk
+ ((continuous_snd.comp continuous_fst).mul (continuous_snd.comp continuous_snd))⟩
+
+attribute [instance] prod.has_continuous_mul
```

Comment:
This line should not be needed, declaring the instance above has just done that.

Reply:
This was there before with `topological_monoid` I just changed the name. I do not know if whoever wrote this line had a reason to do it, but I will try removing it as you suggest.